.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 cache: bundler
 before_script:
-- docker run -d -p 8000:8000 amazon/dynamodb-local
+- bundle exec rake db:create db:migrate
 services:
-- docker
+- postgresql
```

Comment:
Do we even need postgres for this one?

Current spec is no persistance?

Reply:
I'm afraid the conversations I had on Friday evening / Saturday morning implied that's not the case - people will want to be able to do "business intelligence" things (e.g. how many people said "Yes" to question 4). Don't see how we could do that without some data storage.

Reply:
yeah ok so the biz intelligence stuff... 🤔